webpack/published-js-components.js

```diff
@@ -4,7 +4,7 @@ const publishedJSComponents = {
   Box: "components/Box/Box.jsx",
   AccordionItem: "components/Accordion/AccordionItem/AccordionItem.jsx",
   Accordion: "components/Accordion/Accordion/Accordion.jsx",
-  TextWithHighlight: "components/TextWithHighlight/TextWithHighlight.jsx",
+  TextWithHighlight: "components/TextWithHighlight/TextWithHighlight.tsx",
```

Comment:
if its converted please move it to published-ts-components without the .tsx extension

Reply:
Fixed